app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="crayons-card p-5">
+  <% if page.image %>
+    <div class="s:max-w-50 mb-4">
+      <img src="<%= page.image %>" height="auto" alt="Avatar">
+    </div>
+  <% end %>
+  <div class="color-primary">
+    <p class="fs-xl fw-bold lh-tight mb-2">
+      <%= page.title %>
+    </p>
+    <p class="truncate-at-2 mb-2">
+      <%= page.description %>
+    </p>
+  </div>
+  <div class="color-secondary fs-s">
+    <% if page.properties["og:site_name"] %>
```

Comment:
if `page.properties["go:site_name"]` is an array, then this will always be true, even if empty.

My preference would be for us to have a method on OpenGraph for site name.  This would mean we could expose `page.site_name?` and `page.site_name` and reduce the knowledge about the implementation details of "is this an array or is it a single value".

Reply:
Thanks Jeremy. Please view updated code 👍🏾